packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -1425,6 +1432,18 @@ class _SelectableRegionContainerDelegate extends MultiSelectableSelectionContain
 /// This class optimize the selection update by keeping track of the
 /// [Selectable]s that currently contain the selection edges.
 abstract class MultiSelectableSelectionContainerDelegate extends SelectionContainerDelegate with ChangeNotifier {
+  /// Creates a [MultiSelectableSelectionContainerDelegate].
+  ///
+  /// The [selectionSeparator] is used to separate the text of each selectable in
+  /// [selectables].
+  MultiSelectableSelectionContainerDelegate({
```

Comment:
If selectionSeparator then this is not needed

Reply:
Can you please elaborate on what you want done here?

Reply:
I don't remember my previous comment as well, the code looks fine now